components/breadcrumbs/breadcrumb.js

```diff
@@ -0,0 +1,101 @@
+import '../icons/icon.js';
+import '../link/link.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class Breadcrumb extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			_compact: {
+				type: Boolean,
+				reflect: true
+			},
+			href: {
+				type: String,
+				reflect: true
+			},
+			target: {
+				type: String,
+				reflect: true
+			},
+			text: {
+				type: String,
+				reflect: true
+			},
+			ariaLabel: {
+				attribute: 'aria-label',
+				type: String,
+				reflect: true
+			}
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				align-items: center;
+				display: inline-flex;
+			}
+
+			.d2l-breadcrumb-wrapper[data-compact] {
+				display: flex;
+				flex-direction: row-reverse;
+				align-items: center;
+			}
+
+			d2l-icon {
+				height: 8px;
+				padding-left: 10px;
+				padding-right: 5px;
+				width: 8px;
+			}
+
+			:host([dir="rtl"]) d2l-icon {
+				padding-left: 5px;
+				padding-right: 10px;
+			}
+
+			.d2l-breadcrumb-wrapper[data-compact] d2l-icon {
+				padding-right: 10px;
+				padding-left: 0;
+			}
+
+			:host([dir="rtl"]) .d2l-breadcrumb-wrapper[data-compact] d2l-icon {
+				padding-right: 0;
+				padding-left: 10px;
+			}
+		`;
+	}
+
+	constructor() {
+		super();
+		this.href = '#';
+		this._compact = false;
+		this.text = '';
+	}
+
+	firstUpdated() {
```

Comment:
So it won't work from the constructor as it's not attached to the DOM, but what about `connectedCallback`? `firstUpdated` is OK, but this means it'll render twice.

Reply:
`connectedCallback` seems to be working fine. I was testing it bad I guess with console.log

Reply:
It does work. I was just using console.log to track changes which did not work properly. (Sorry)